app/assets/javascripts/listings/components/eligibilitySection.js.coffee

```diff
@@ -12,8 +12,10 @@ angular.module('dahlia.components')
       @error = ListingPreferenceService.error
 
       @occupancy = (unitSummary) ->
-        return '1' if unitSummary.maxOccupancy == 1
-        unitSummary.minOccupancy + '-' + unitSummary.maxOccupancy
+        if unitSummary.minOccupancy == 1 || unitSummary.maxOccupancy == null
```

Comment:
I think this logic is still wrong

### If max Occupancy is not null
We want
- "1 person" if min and max occupancy == 1
- "1-2 people" (minOccupancy-maxOccupancy) if min and max occupancy are both defined

### If max occupancy is null 
We want
- "at least 1 person" if minOccupancy is 1
- "at least 3 people" if minOccupancy > 1

Reply:
@akegan Thanks for this!